source/dagger/gameplay/plight/plight.cpp

```diff
@@ -26,6 +29,9 @@ struct PlightCharacter
     Animator& animator;
     InputReceiver& input;
     TopdownCharacter& character;
+    PlightCollision& col;
+    Transform& transform;
+    CombatStats& cstats;
 
     static PlightCharacter Get(Entity entity)
```

Comment:
Styleguide: parameters should always have a tracing underscore, eg. `Entity entity_`